premake5.lua

```diff
@@ -151,14 +151,6 @@ newoption {
      default     = "true",
      allowed     = { { "true", "True"}, { "false", "False" } }
   }
- 
```

Comment:
Since this is still a very experimental featur, can we keep this flag and have the heterogeneous-hello-world example behind this flag?

Reply:
Sounds good, I can add it back in the next commit. Also have to clean the file structure and generate the new solution.